cmd/packages.go

```diff
@@ -134,6 +134,11 @@ func setPackageFlags(flags *pflag.FlagSet) {
 		"exclude paths from being scanned using a glob expression",
 	)
 
+	flags.StringArrayP(
+		"enable-cataloger", "", nil,
```

Comment:
this could probably just be `catalogers` on the cli.

Reply:
I was thinking the same thing here 👍